test/compliance/CREATE.test.js

```diff
@@ -25,7 +25,9 @@ describe('CREATE', () => {
   // Load model before test suite to generate test suite from model definition
   const model = cds.load(__dirname + '/resources/db', { sync: true })
 
-  const literals = Object.keys(model.definitions).filter(n => model.definitions[n].kind === 'entity')
+  const literals = Object.keys(model.definitions)
+    .filter(n => model.definitions[n].kind === 'entity')
+    .filter(n => n !== "complex.JoinBooksAndAuthors" )
```

Comment:
It seems like you are trying to exclude the more complex view. Which this test suite does not (yet) support. It would be good to make the check less specific and also catch any entities added in the future which would cause trouble for this test.

You would have to try it out, but I guess it should look something like:

```suggestion
    .filter(n => !model.definitions[n].query) // Exclude views
    .filter(n => !model.definitions[n].query?.from?.args) // Exclude views with join source
```